### PR TITLE
Initial version of a new osg-notify tool.

### PR DESCRIPTION
**Pull request by bbockelm**
Rough draft of the new notification tool.

### CODE REVIEW

**Comment by matyasselmeci** [comment]
Should have some sort of a `--dry-run` option that only prints the message to console.

**Comment by matyasselmeci** [comment]
This should also be integrated with a tool that queries contacts from the osg-topology, such that we can do something like `osg-notify --type security --service ce ...` to a) send a message to security contacts for CEs and b) label it as coming from the Security Team.

**Comment by bbockelm** [comment]
@matyasselmeci - now it works like this:

```
osg-notify --type test \
   --message /tmp/baz \
   --subject 'This is a test osg-notify email' \
   --dry-run \
   --oim-recipients resources \
   --oim-contact-type security \
   --from security
```

Thoughts?

File: src/topology_utils.py [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+        vo_info = {}
+        for child_info in child_vo:
+            vo_info[child_info.tag] = child_info.text
+        if 'ID' in vo_info and 'Name' in vo_info:
+            vo_map[vo_info['Name'].lower()] = vo_info['ID']
+
+    return vo_map
+
+
+SERVICE_IDS = {'ce': 1,
```
Comment (matyasselmeci) [inline src/topology_utils.py]: You can get service IDs from `topology/services.yaml`

**Comment by jteheran** [comment]
It looks good overall. 
We can start sending announcements right away!

File: src/topology_utils.py (hunk 1 above)
Comment (bbockelm) [reply src/topology_utils.py]: Hm.  I'd prefer to keep this PR to refactoring.  I'll open a new issue for this idea.

**Pull request merged**